src/Calculator/Controls/HorizontalNoOverflowStackPanel.cpp

```diff
@@ -57,12 +59,14 @@ Size HorizontalNoOverflowStackPanel::ArrangeOverride(Size finalSize)
         {
             // stack the items horizontally (left to right)
             item->Arrange(Rect(posX, 0, itemWidth, finalSize.Height));
+            AutomationProperties::SetAccessibilityView(item, ::AccessibilityView::Control);
```

Comment:
[Content is the default](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.accessibilityview) for accessibility view. Can you double check this with GetAccessibilityView or some other way and see what it was set before we set it to raw?

Reply:
The accessibility view was indeed content when visible originally. I've changed that line to set it to content when visible instead of control.